packages/entities/entities-upstreams-targets/src/components/TargetForm.cy.ts

```diff
@@ -51,10 +51,10 @@ describe('<TargetForm />', { viewportHeight: 700, viewportWidth: 700 }, () => {
       cy.get('.kong-ui-entities-target-form').should('be.visible')
       cy.get('.kong-ui-entities-target-form form').should('be.visible')
       // button state
-      cy.get('.k-prompt-cancel').should('be.visible')
-      cy.get('.k-prompt-proceed').should('be.visible')
-      cy.get('.k-prompt-cancel').should('be.enabled')
-      cy.get('.k-prompt-proceed').should('be.disabled')
+      cy.get('button[data-testid="modal-cancel-button"]').should('be.visible')
+      cy.get('button[data-testid="modal-action-button"]').should('be.visible')
+      cy.get('button[data-testid="modal-cancel-button"]').should('be.enabled')
+      cy.get('button[data-testid="modal-action-button"]').should('be.disabled')
```

Comment:
Same comment on selectors; can we update any/all of these with `cy.getTestId` instead?

Reply:
Updated